code/network/multi_voice.cpp

```diff
@@ -2076,7 +2076,7 @@ void multi_voice_test_process_next_chunk()
 void multi_voice_test_record_start()
 {
 	// if there is test recording going on already, don't do anything
-	if(Multi_voice_test_record_stamp != -1){
+	if ( !Multi_voice_test_record_stamp.isValid() ) {
```

Comment:
I don't think the logical NOT is correct here.

Reply:
Yep, I got that wrong in a few places but must have missed this one when going back to fix them.